.redhat-ci.yml

```diff
@@ -135,3 +135,23 @@ tests:
 
 artifacts:
   - test-suite.log
+
+---
+
+inherit: false
+branches:
+    - master
+    - auto
+    - try
+
+context: f25-flatpak
+required: false
+
+# This test case wants an "unprivileged container with bubblewrap",
+# which we don't have right now; so just provision a VM and do a
+# docker --privileged run.
+host:
+  distro: fedora/25/atomic
+
+tests:
+  - docker run --rm --privileged -v $(pwd):/srv/code registry.fedoraproject.org/fedora:25 /bin/sh -c "cd /srv/code && ./ci/flatpak.sh"
```

Comment:
Can we also upload the test results?

```yaml
artifacts:
  - test-suite.log
```

And in the `flatpak.sh` script, have something like:

```sh
trap '[ ! -f test-suite.log ] || cp test-suite.log /srv/code' ERR
```

?